app/controllers/ApplicationController.scala

```diff
@@ -320,6 +320,13 @@ class ApplicationController @Inject() (implicit val env: Environment[User, Sessi
     }
   }
 
+  /**
+    * Returns the maintenance page.
+    */
+def maintenance = UserAwareAction.async { implicit request =>
+  Future.successful(Ok(views.html.maintenance("Project Sidewalk - Maintenance")))
```

Comment:
Wait, why did you remove the logging?

Reply:
Ahhhhhh oh my god, I was stupid. Just fixed everything now, hopefully. @misaugstad